### DISCUSSION TITLE
Schedule E-mail Forwarding

### DISCUSSION THREAD

**Discussion by davidmarkjohnston** [simple-login/app#1583] [Feature Requests]
Add a feature to schedule when e-mails are forwarded (eg 09:00 each day, 09:00 each Monday etc)

**Comment by t3dium** [comment]
this’d be a great feature to have, especially since some email providers (tutanota for e.g) don’t natively support the feature, and protonmail’s paywalled the feature (or at least the ability to customise times)..
